tensorflow/python/client/virtual_gpu_test.py

```diff
@@ -206,8 +209,6 @@ def testStatsContainAllDeviceNames(self):
       # adding virtual devices in the future, thus must be called within a
       # context of a session within which virtual devices are created. Same in
       # the following test case.
-      if not test.is_gpu_available(cuda_only=True):
```

Comment:
Probably doesn't matter for this fork, but it would be good to ensure at least one DML device is found before trying to run this. :)